modules/@apostrophecms/ui/ui/apos/components/AposTreeRows.vue

```diff
@@ -162,13 +164,39 @@ export default {
       return true;
     }
   },
+  mounted() {
+    // Use $nextTick to make sure attributes like `clientHeight` are settled.
+    this.$nextTick(() => {
+      const branches = this.$el.querySelectorAll('[data-apos-tree-branch]');
+
+      branches.forEach(branch => {
+        // Add padding to the max-height to avoid needing a `resize`
+        // event listener updating values.
+        const height = branch.clientHeight + 20;
+        branch.setAttribute('data-apos-tree-branch', `${height}px`);
+        branch.style.maxHeight = `${height}px`;
+      });
+    });
+  },
   methods: {
     startDrag() {
       this.$emit('busy', true);
     },
     endDrag(event) {
       this.$emit('update', event);
     },
+    toggleSection(event) {
```

Comment:
This probably also could leverage the `ref` if used.

Reply:
Same here. We don't know the level that the item is on, and it's all generated, so I'm not sure how that works.